recipes/openvino/all/conanfile.py

```diff
@@ -45,7 +45,9 @@ class OpenvinoConan(ConanFile):
         "enable_tf_frontend": [True, False],
         "enable_tf_lite_frontend": [True, False],
         "enable_paddle_frontend": [True, False],
-        "enable_pytorch_frontend": [True, False]
+        "enable_pytorch_frontend": [True, False],
+        # Threading
+        "threading": ["tbb", "omp"]
```

Comment:
- `seq` is also available to disable the threading 
- can we enable `threading` since 2024.4 ? because before that version OpenMP support was not enough tested / supported.